flash/__init__.py

```diff
@@ -26,7 +26,7 @@
     from flash.core.data.input_transform import InputTransform
```

Comment:
Let's import the new one here
```suggestion
    from flash.core.data.io.input_transform import InputTransform
```